windows/en-US/firefox.adml

```diff
@@ -23,6 +23,10 @@
       <string id="Allow">Allowed Sites</string>
       <string id="Block">Blocked Sites</string>
       <string id="Authentication_SPNEGO">SPNEGO</string>
+      <string id="AppUpdateURL">URL for Firefox Updates</string>
+      <string id="AppUpdateURL_Explain">If this policy is enabled, you can set a URL to an other Updateserver than the default. This could be helpful, if you run your own Update-Server in your Network and the Clients have no direct access to the Internet and the Mozilla Update Server.
```

Comment:
Can you add a space and take out the capital on Update server?

Reply:
It is "update server" not Updateserver or Update-Server, right?